app/assets/javascripts/3dbio_viewer/package.json

```diff
@@ -5,9 +5,10 @@
     "dependencies": {
         "@3dbionotes/pdbe-molstar": "3.1.0-beta.1",
         "@3dbionotes/protvista-pdb": "2.0.1-beta.9",
-        "@material-ui/core": "^4.11.2",
+        "@material-ui/core": "latest",
```

Comment:
@p3rcypj I am aware we have to go to the latest material-ui versions for the new graphical component to work properly but can we maybe pin a particular version instead of setting it to "latest"?

Reply:
It is because of the `/lab` dependency. The lab depends on the material-ui core and when the core isn't the expected version it throws errors that people on SO reply as: "they broke it again". In this situation and knowing that `@material-ui` no longer receives active development as they shout at [npm](https://www.npmjs.com/package/@material-ui/lab), in favor of `@mui`, I think that they will not publish a version without backwards compatibility. That's the reason I changed it to "latest", but please let me know your opinion @tokland

Reply:
@adrianq Can we pin the latest version that we have of both dependencies?

Reply:
@p3rcypj yes!